build.sbt

```diff
@@ -8,7 +8,7 @@ bucketSuffix  := "era7.com"
 
 libraryDependencies ++= Seq(
   "ohnosequences"         %% "cosas"        % "0.8.0",
-  "com.github.pathikrit"  %% "better-files" % "2.13.0",
+  "com.github.pathikrit"  %% "better-files" % "2.15.0",
```

Comment:
??????????? :flushed:
I all support the update, but not like change-on-the-way, because this makes this incompatible with _everything_ else. Please, release the fixes or features first without this.